src/Entity/Academy.php

```diff
@@ -19,6 +20,12 @@ class Academy
      */
     private $programs;
 
+    /**
+     * One Academy has many Cities. This is the inverse side.
+     * @OneToMany(targetEntity="City", mappedBy="academy", cascade={"remove"})
+     */
+    private $cities;
```

Comment:
Will the Academy has cities? Or maybe the program? or eve program Event will have a city? :)

Reply:
My idea was that each Academy can have multiple cities, so by linking two columns, I can get cities info to Kristina and she could use that info inside single academy card.

Reply:
Should you fetch cities by looking though programs of academy?
Like searching for academy, then related academy programs, and then collecting all the cities of programs?
Once city will change of the program, will you also change the available cities of a Academy?

Reply:
Neither Program nor City entities have anything connecting them between, so I had to figure out a way to grab City info from Academy.